src/pages/Donate/Donate.jsx

```diff
@@ -16,7 +16,7 @@ const Donate = () => {
 
   const [clubData, setClubData] = useState([]);
   const [loading, setLoading] = useState(false);
-  const [isLoggedIn] = useState(Cookies.get("token"));
+  const [isLoggedIn] = useState(Cookies.get("Token"));
```

Comment:
Again, I think it would be better to remove the states all through out.

Reply:
Removed the states and implemented with Cookies.